pkg/service/restore/config.go

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2023 ScyllaDB
+
+package restore
+
+import (
+	"github.com/pkg/errors"
+	"github.com/scylladb/scylla-manager/v3/pkg/service"
+)
+
+// Config specifies the backup service configuration.
+type Config struct {
+	DiskSpaceFreeMinPercent   int `yaml:"disk_space_free_min_percent"`
+	LongPollingTimeoutSeconds int `yaml:"long_polling_timeout_seconds"`
+}
+
+func DefaultConfig() Config {
+	return Config{
+		DiskSpaceFreeMinPercent:   10,
+		LongPollingTimeoutSeconds: 10,
```

Comment:
What if `LongPollingTimeoutSeconds` is < 0 ? Should it be validated ? 
Looks that the value is passed to rclone in the api call, but I'm curious how RClone interprets < 0.